src/components/Budget/BudgetView/BudgetViewContainer.js

```diff
@@ -57,6 +60,15 @@ const BudgetViewContainer = ({ history, resources }) => {
 
   const lastMenu = (
     <PaneMenu>
+      <Button
+        marginBottom0
+        buttonStyle="primary"
+        data-test-add-transfer-button
+        onClick={() => toggleTransferModal(true)}
```

Comment:
https://github.com/folio-org/stripes-acq-components/blob/master/lib/hooks/useModalToggle.js#L7

no arguments, it's toggle, no need new function